src/lwip/silabs/lwipopts-siwx.h

```diff
@@ -54,7 +54,7 @@
 #define LWIP_DHCP_AUTOIP_COOP (0)
 #define LWIP_SOCKET_SET_ERRNO 0
 #define IP_REASS_MAX_PBUFS 0
-#define IP_REASSEMBLY 1
+#define IP_REASSEMBLY 0
```

Comment:
i thought this was done as part of 2.7.0-1.4 only... Isnt it?

Reply:
This is for the IPv4, it was disabled for the IPv6 in 2.7.0